docs/platform/linode-gpu/getting-started-with-gpu/index.md

```diff
@@ -113,26 +143,46 @@ Installation on CentOS and RHEL operating systems will fail following this as th
 +-----------------------------------------------------------------------------+
 {{< /output >}}
 
-    In the output above, we can see that our Driver is installed and functioning correctly, the version of CUDA attributed to it, and other useful statistics.
+    In the output above, we can see that our driver is installed and functioning correctly, the version of CUDA attributed to it, and other useful statistics.
 
 ## Manual Install
 
-This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can be completed similarly on a Distro of your choice if needed:
+This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can also be completed similarly on another distro of your choice if needed:
+
+1. Visit NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+
+1. Make sure that the options from the drop-down menus reflect the following values:
```

Comment:
Added a text table here for the values (in addition to the screenshot). A blind reader wouldn't be able to see the values in the screenshot